libs/api-sdk/src/index.test.ts

```diff
@@ -217,22 +217,61 @@ describe("Bandada API SDK", () => {
                 expect(res).toBeUndefined()
             })
         })
-        describe("#removeMemberByApiKey", () => {
-            it("Should remove a member from a group using an API Key", async () => {
+        describe("#addMembers", () => {
+            it("Should add multiple members to the group using an API Key", async () => {
                 requestMocked.mockImplementationOnce(() => Promise.resolve())
 
                 const groupId = "10402173435763029700781503965100"
-                const memberId = "1"
+                const memberIds = ["1", "2", "3"]
                 const apiKey = "70f07d0d-6aa2-4fe1-b4b9-06c271a641dc"
 
                 apiSdk = new ApiSdk(SupportedUrl.DEV)
-                const res = await apiSdk.removeMemberByApiKey(
+                const res = await apiSdk.addMembersByApiKey(
                     groupId,
-                    memberId,
+                    memberIds,
                     apiKey
                 )
                 expect(res).toBeUndefined()
```

Comment:
Maybe we can check the group to see if the members have been added as expected? Only this expect is a bit lax expecting an `undefined` (which means everything and nothing in JS lmao).

Reply:
Here is testing that the sdk function is working and not that the api is working correctly, what would you change in this test?

Reply:
Hmm if we expect that the SDK is working correctly and the constraint by which we define the expected result an execution without any error, that's totally fine! <3